app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -562,4 +519,29 @@ class PatientSummaryScreenController @Inject constructor(
 
     return durationSinceBpCreated <= bpEditableFor
   }
+
+  @WorkerThread
```

Comment:
Curious, does this throw a runtime exception if invoked from the main thread?

Reply:
Nope, it's just an annotation to Android Lint which will show IDE warnings if the method is inferred to be running on the main thread.